static-site/src/components/sars-cov-2/builds.jsx

```diff
@@ -75,7 +74,7 @@ class Index extends React.Component {
         <splashStyles.FocusParagraph>
           This section is an index of public Nextstrain builds (datasets) for SARS-CoV-2, organized by geography.
           Some of these builds are maintained by the nextstrain team and others are maintained by independent research groups.
-          (<Link to="docs/bioinformatics/what-is-a-build">See here</Link> for more information on what a build is, and see <a href="https://nextstrain.github.io/ncov/">this tutorial</a> for a walkthrough of running your own phylogenetic analysis of SARS-CoV-2 data!)
+          (<a href="https://docs.nextstrain.org/projects/augur/en/migrate-docs/faq/what-is-a-build.html" >See here</a> for more information on what a build is, and see <a href="https://nextstrain.github.io/ncov/">this tutorial</a> for a walkthrough of running your own phylogenetic analysis of SARS-CoV-2 data!)
```

Comment:
Leaving a placeholder "Todo" here as this should be /latest but this requires the Augur PR to be merged